roles/ara_api/tasks/database_engine/django.db.backends.postgresql.yaml

```diff
@@ -42,7 +42,7 @@
   pip:
     # Pin psycopg2 until we upgrade to django 3.2 LTS
     # https://github.com/ansible-community/ara/issues/320
-    name: psycopg2<2.9
+    name: psycopg2<3
```

Comment:
This made me go down quite a rabbithole -- django is currently allowed to be up to 4.2 which is the latest LTS: https://github.com/ansible-community/ara/blob/8eda9c87086a289cf9bf41a2d4cc4060e2d50ad7/setup.cfg#L67-L69

I suppose we are past April 2024 so we could bump the version of django to 4.2 and figure out whether we can use psycopg3. I will not go down this hole just yet but I'll make sure ``<3`` works.

Reply:
To be clear, we removed the pinning on psycopg2 in a separate patch here: https://github.com/ansible-community/ara-collection/commit/7dc61fe034d691a8184a8df76e26c8f64380e0b9

I checked and the last version is currently 2.9.9:
![image](https://github.com/ansible-community/ara-collection/assets/1291204/27813c11-1627-4ec5-961a-22b02e070795)

Pinning it to ``<3`` seems fair and it looks like a :heart: